src/SceneView.tsx

```diff
@@ -28,8 +38,16 @@ export default function SceneView(props: Props) {
       setParams: (params: object, target?: TargetRoute<string>) => {
         helpers.setParams(params, target ? target : { key: route.key });
       },
+      setOptions: (options: object) =>
+        setOptions(o => ({
```

Comment:
AFAIK we don't want to have setOptions. If we can set it after mounting, why not to use param?